tools/generator-bot-template-generator/app/templates/generators/app/templates/.gitignore

```diff
@@ -0,0 +1,2 @@
+# prevent appsettings.json get checked in
+**/appsettings.json
```

Comment:
Not sure if I have the right context but appsettings.json should be OK to check in but we need to support appsettings.Development.json and have that one ignored for local testing.

Reply:
Good catch! I have removes that. This is the gitignore file that will be bootstrapped in the end project for the user. Its mainly empty as its up to them to what they want to ignore but they would presumable not want to gitignore appsettings. I have fixed this.